python/test/unit/runtime/test_hipblas.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+import torch
+from triton._internal_testing import is_hip, is_hip_cdna3, is_hip_cdna4
+
+
+@pytest.mark.parametrize("m, n, k", [(16, 16, 16), (32, 16, 16), (16, 32, 16), (16, 16, 32)])
+@pytest.mark.parametrize("dtype_str", ["float8_e4m3fnuz", "float16"])
+def test_hipblas(m, n, k, dtype_str, device):
```

Comment:
Let's not duplicate. Instead we can use some if-else to support hipblas in the existing `test_cublas.py` test (and rename properly).